node/src/storage/triple_storage.rs

```diff
@@ -14,6 +14,25 @@ use std::collections::{HashMap, HashSet};
 use std::sync::Arc;
 use tokio::sync::RwLock;
 
+pub struct TripleKey<'a> {
+    pub account_id: &'a str,
+    pub triple_id: TripleId,
+    pub mine: bool,
+}
+
+impl Keyable for TripleKey<'_> {
+    fn key(&self) -> Key {
+        Key {
+            path: Some(vec![PathElement {
+                kind: Some(TripleData::kind()),
+                name: Some(format!("{}/{}", self.account_id, self.triple_id)),
```

Comment:
Should we add mine here?

Reply:
nope, mine does not need to be a part of the entity key. All triples are stored regardless of whether they belong to the node or not, where we have another field for `mine: bool`. This here is the key in reference of that entity, so it's not needed to store this value in the key